app/build.gradle

```diff
@@ -105,23 +105,23 @@ android {
 dependencies {
 	implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
 
-	implementation 'com.google.android.material:material:1.2.1'
-	implementation 'androidx.appcompat:appcompat:1.2.0'
-	implementation 'androidx.activity:activity:1.1.0'
+	implementation 'com.google.android.material:material:1.3.0'
+	implementation 'androidx.appcompat:appcompat:1.3.0'
+	implementation 'androidx.activity:activity:1.2.3'
 	implementation('androidx.fragment:fragment:1.1.0!!') {
```

Comment:
I wonder if those two (new activity, but old fragment library) actually play well together, but I guess we'll find out.

Reply:
At least there were no obvious problems while playing a bit with the app.